x-pack/plugins/watcher/public/components/chart_tooltip/_chart_tooltip.scss

```diff
@@ -0,0 +1,21 @@
+.chartTooltip {
```

Comment:
@bmcconaghy @jen-huang Either of you know where to test this in the UI?

Reply:
Thanks for the review @cjcenizal . You can see the chart tooltip when creating a threshold alert:

<img width="907" alt="screenshot 2018-10-02 15 27 40" src="https://user-images.githubusercontent.com/446285/46374998-d4d54080-c657-11e8-9681-b2be09717cc7.png">